CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v37.0.0...main)
 
+* Rust
+  * Don't return a result from `submit_ping`. The boolean result indicates whether a ping was submitted ([#1613](https://github.com/mozilla/glean/pull/1613))
```

Comment:
```suggestion
  * Don't return a result from `submit_ping`. The boolean return value indicates whether a ping was submitted ([#1613](https://github.com/mozilla/glean/pull/1613))
```

(the most minor of nits, and only because you're using "result" in the first sentence to mean `Result` and using the same word with a different meaning in the second sentence)